android/assets/jsons/translations/Portuguese.properties

```diff
@@ -722,97 +722,97 @@ You have won a scientific victory! = Você ganhou uma vitória ciêntifica!
  # Requires translation!
-A puppeted city can be annexed at any time. = 
+A puppeted city can be annexed at any time. = Uma cidade fantoche pode ser anexada a qualquer momento.
  # Requires translation!
-Liberate = 
+Liberate = Liberar
  # Requires translation!
-Liberating a city returns it to its original owner, giving you a massive relationship boost with them! = 
+Liberating a city returns it to its original owner, giving you a massive relationship boost with them! = Liberar uma cidade devolve ela a seu dono original, dando um estímulo gigantesco de relacionamento com ele!
  # Requires translation!
-Raze = 
+Raze = Arrasar
  # Requires translation!
-Razing the city annexes it, and starts razing the city to the ground. = 
+Razing the city annexes it, and starts razing the city to the ground. = Arrasar uma cidade a anexa, e começa a arrasa-la às cinzas.
  # Requires translation!
-The population will gradually dwindle until the city is destroyed. = 
+The population will gradually dwindle until the city is destroyed. = A população ira gradualmente diminuir até que a cidade seja destruída.
  # Requires translation!
-Destroy = 
+Destroy = Destruir
  # Requires translation!
-Destroying the city instantly razes the city to the ground. = 
+Destroying the city instantly razes the city to the ground. = Destruir a cidade arrasa instantaneamente a cidade às cinzas.
  # Requires translation!
-Remove your troops in our border immediately! = 
+Remove your troops in our border immediately! = Remova suas tropas das nossas fronteiras imediatamente!
 Sorry. = Desculpe.
 Never! = Nunca!
 
  # Requires translation!
-Offer Declaration of Friendship ([30] turns) = 
+Offer Declaration of Friendship ([30] turns) = Oferecer Declaração de Amizade ([30] turnos)
  # Requires translation!
-My friend, shall we declare our friendship to the world? = 
+My friend, shall we declare our friendship to the world? = Meu amigo, deveríamos declarar nossa maizade ao mundo?
  # Requires translation!
-Sign Declaration of Friendship ([30] turns) = 
+Sign Declaration of Friendship ([30] turns) = Assinar Declaração de Amizade ([30] turnos)
 We are not interested. = Nós não estamos interessados.
  # Requires translation!
-We have signed a Declaration of Friendship with [otherCiv]! = 
+We have signed a Declaration of Friendship with [otherCiv]! = Nós assinamos Declaração de Amizade com [otherCiv]!
  # Requires translation!
-[otherCiv] has denied our Declaration of Friendship! = 
+[otherCiv] has denied our Declaration of Friendship! = [otherCiv] negou nossa Declaração de Amizade!
 
 Basics = Básicos
 Resources = Recursos
 Terrains = Terrenos
  # Requires translation!
-Tile Improvements = 
+Tile Improvements = Melhorias de Ladrilhos
 Unique to [civName], replaces [unitName] = Exclusivo da civilização [civName], substitui [unitName]
  # Requires translation!
-Unique to [civName] = 
+Unique to [civName] = Unico ao(à) [civName]
  # Requires translation!
-Tutorials = 
+Tutorials = Ttoriais
```

Comment:
u?

Reply:
You are right again,
should be 'Tutoriais'